pom.xml

```diff
@@ -139,6 +139,7 @@
     <slf4j.version>1.7.2</slf4j.version>
     <test.output.redirect>true</test.output.redirect>
     <hadoop-openstack.version>2.6.0</hadoop-openstack.version>
+    <surefire.useSystemClassLoader>true</surefire.useSystemClassLoader>
```

Comment:
The default has to be true, otherwise you will see errors like this:
https://amplab.cs.berkeley.edu/jenkins/job/Tachyon-Pull-Request-Builder/8544/org.tachyonproject%24tachyon-core-client-internal/testReport/